test/e2e/config/multikueue/worker/patch_crds.yaml

```diff
@@ -0,0 +1,5 @@
+$patch: delete
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: mpijobs.kubeflow.org
```

Comment:
Here is the duplicate of `test/e2e/config/multikueue/manager/patch_crds.yaml`. We can move it to common directory.

Reply:
I am sorry, we can't kustomize requires to have patches in the same folder or in the folder below